src/librustc_typeck/error_codes.rs

```diff
@@ -4484,7 +4484,7 @@ let _ = (2.0 as f32).neg();
 
 E0690: r##"
 A struct with the representation hint `repr(transparent)` had zero or more than
```

Comment:
This error code is longer used only for structs, so it should be generalized.

Likewise for E0691 below.